Schemas_draft-03/Collaboration/Viewpoint/viewpoint_GET.json

```diff
@@ -39,5 +39,14 @@
         "snapshot": {
             "$ref": "snapshot_GET.json"
         }
+	    "authorization": {
```

Comment:
You have mixed tabs and spaces and a comma is missing

Reply:
I've just realised I asked to merge this change with the wrong branch. I've fixed the branch to `release_2_2`. Please check if this issue exists in the 2_2 branch and raise a new pull request to fix it there. Thanks